app/views/account/profiles/edit.html.slim

```diff
@@ -2,4 +2,6 @@
 .container.has-text-centered
   .column.is-4.is-offset-4.box
     .title My Profile Settings
+    figure.image.is-64x64.container
```

Comment:
i believe correct chain of classes would be `.container.image.is-64x64`

Reply:
I don't think .container is needed at all?
container is a "container", for columns, etc. I don;t know if this is how bulma works